### DISCUSSION TITLE
update XTIDE Universal BIOS to r624

### DISCUSSION THREAD

**Discussion by telanus** [86Box/86Box#2937] [Ideas]
Could the xtide roms be updated to r624 https://www.xtideuniversalbios.org/binaries/r624/?

**Comment by lemondrops** [comment]
This is now done in 86Box/roms#192.

**Comment by roytam1** [comment]
what about updating to [r625](https://www.xtideuniversalbios.org/changeset/625/xtideuniversalbios)?
https://www.xtideuniversalbios.org/binaries/r625/

**Comment by OBattler** [comment]
Closed since it has now been fulfilled.
